### DISCUSSION TITLE
Implement Electron `safeStorage` api

### DISCUSSION THREAD

**Discussion by patrickhoogkamer** [NativePHP/laravel#287] [Ideas]
Currently there is no way to safely store certain values, encryption keys for example, in the user's keychain (or platform equivalent when not on macOS). 

Electron has an api called [safeStorage](https://www.electronjs.org/docs/latest/api/safe-storage) that could be implemented in NativePHP to accomplish this.

**Comment by simonhamp** [comment]
This just got released! [Docs here](https://nativephp.com/docs/1/the-basics/system#encryption-decryption)
